woo-razorpay.php

```diff
@@ -17,6 +17,7 @@
 
 require_once __DIR__.'/includes/razorpay-webhook.php';
 require_once __DIR__.'/razorpay-sdk/Razorpay.php';
+require_once(ABSPATH.'wp-admin/includes/plugin.php');
```

Comment:
Could you remove the parentheses for consistency with the above imports?

Reply:
yep